# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/PCookieStore.ipdl
# Commit: 6a36e7a65173
# Full Hash: 6a36e7a65173113ea12080e630530655c190c0ab
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-13 04:28:43
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 10 - CookieStore API - CHIP implementation shared between Document and CookieStore, r=cookie-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D220288
# ==============================================================================

diff -r 2fbb65a885f6 -r 6a36e7a65173 dom/cookiestore/PCookieStore.ipdl
--- a/dom/cookiestore/PCookieStore.ipdl	Thu Sep 12 16:48:22 2024 +0000
+++ b/dom/cookiestore/PCookieStore.ipdl	Thu Sep 12 16:48:23 2024 +0000
@@ -32,6 +32,10 @@
 parent:
   async GetRequest(nsString domain,
                    OriginAttributes attrs,
+                   OriginAttributes? partitionedAttrs,
+                   bool thirdPartyContext,
+                   bool partitionForeign,
+                   bool usingStorageAccess,
                    bool matchName,
                    nsString name,
                    nsCString path,
@@ -39,6 +43,9 @@
 
   async SetRequest(nsString domain,
                    OriginAttributes attrs,
+                   bool thirdPartyContext,
+                   bool partitionForeign,
+                   bool usingStorageAccess,
                    nsString name,
                    nsString value,
                    bool session,
@@ -50,6 +57,9 @@
 
   async DeleteRequest(nsString domain,
                       OriginAttributes attrs,
+                      bool thirdPartyContext,
+                      bool partitionForeign,
+                      bool usingStorageAccess,
                       nsString name,
                       nsString path,
                       bool partitioned,